.github/workflows/update-conda-lockfile.yml

```diff
@@ -4,17 +4,16 @@ name: update-conda-lockfile
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 9 * * 1" # Mondays at 9AM UTC
+    - cron: "0 9 * * 1" # Mondays at 9AM UTC/middle of the night in USA.
 
 # What branch does this action run on?
 # - workflow_dispatch: Whatever branch it was run against.
 # - schedule: Always the same branch (will be dev or main)
-# - push: Base branch of the PR.
 
 jobs:
   update-conda-lockfile:
     runs-on: ubuntu-latest
-    if: ${{ (github.event_name == 'push' && github.actor != 'pudlbot') || (github.event_name == 'schedule' && github.repository == 'catalyst-cooperative/pudl') || (github.event_name == 'workflow_dispatch') }}
```

Comment:
I removed all of the on `push` logic, since the `pre-commit` hooks seem to be working well, so this workflow will just be used to ensure our dependencies get updated once a week on a schedule.

Reply:
Sweet. Thanks, I forgot to clean this up when I removed the `push` trigger.